packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -257,12 +256,17 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
 
     data = generatedData.dimensions;
     legendRows = generatedData.legendRowsData;
-    today = generatedData.today;
     scaledWidgetUnit.current = generatedData.unit; // here state doesn't matter, as this is always the latest re-render
+    currentUnit = generatedData.unit;
+    areas = generatedData.areas;
   }
 
   const metricsApiCallError = error?.[0]?.reason;
 
+  const tickFormat =
+    duration.unit === 'min' || duration.unit === 'hr'
```

Comment:
if there is isToday, we can decide this, this will not be good if we have features like 50 hr, 48 hr like that